### ISSUE TITLE
Updates

### ISSUE THREAD

**Issue by JuniorJPDJ** [yupi2/synergy#11]
Pssst..
You gonna maintain this, or not? :<

**Comment by yupi2** [comment]
No. I don't run two OSs simultaneously anymore... so I don't need synergy anymore.

Also I don't think it's worth working on the synergy code.
I'd prefer to use a new alternative with better screen organization, a (Windows) kernel driver for intercepting input, and UDP or QUIC for computer-to-computer.
But then I'd have to make it.
Also something something AGPL because symless
